apps/backend/sample-files/transaction.json

```diff
@@ -12,20 +12,20 @@
     "source": {
       "source_id": "SBC",
       "location_id": 61,
-      "accepted_payment_methods": ["CASH", "CHQ", "P", "M", "V"]
+      "accepted_payment_methods": ["CASH", "CHQ", "P", "M", "V", "US_FUNDS"]
     },
     "payments": [
       {
-        "amount": 10.2,
-        "currency": "CAD",
-        "exchange_rate": 0,
-        "payment_method": "CASH",
+        "amount": 15.0,
+        "adjusted_amount": 20.12,
+        "currency": "USD",
+        "exchange_rate": 1.34,
+        "payment_method": "US_FUNDS",
```

Comment:
is this a new payment method because it exists in GARMS? 

do we need `US_FUNDS`? then is CAD payments `CAD_FUNDS`? 

if we say just CASH,  is it CAD or any currency cash?